### PR TITLE
[HW2] Mikhail Volkhov ITMO M3439

### PR DESCRIPTION
**Pull request by volhovm**
Haven't tested it yet.

Not related; though:
1. I do not understand why do I need to cherry-pick my HW1 solutions. Will you come up with a decent scheme to do it? I suggest every participant to have his own branch instead of fork and present PRs to this branch from (another) feature branch. Though it, of course, requires more permissions, but you can still block `master` and have admin access.
2. It's pretty weird not to have a proper `.gitignore`. Be sure to check this PR: https://github.com/dboulytchev/compiler-workout/pull/107

### CODE REVIEW

**Comment by volhovm** [comment]
Oh, it actually passes all the tests. Nice.

File: src/Embedding.ml [hunk 1]
```diff
@@ -12,7 +12,7 @@ let ( !? ) n = Expr.Const n
 
 (* Implementation of operators *)
 let binop op x y = Expr.Binop (op, x, y)
-
```
Comment (volhovm) [inline src/Embedding.ml]: Also trailing spaces. Can you _please_ set up needed bindings in your editor?

Comment (anlun) [reply src/Embedding.ml]: @dboulytchev

**Comment by anlun** [comment]
Если у Вас есть не связанные с домашкой предложения, то их нужно направлять на почту, а не писать в PR.

**Comment by volhovm** [comment]
Sure, np.

**Comment by anlun** [comment]
Зачтено.

**Pull request closed without merging**